cli/package.json

```diff
@@ -7,7 +7,7 @@
     "clean": "node ./scripts/clean.js",
     "prebuild": "yarn postinstall && node ./scripts/start-build.js",
     "build": "node ./scripts/build.js",
-    "dtslint": "dtslint types",
+    "dtslint": "dtslint --expectOnly --localTs ../node_modules/typescript/lib types",
```

Comment:
I think this will only validate we work on our local TS version (4.x). I think the goal was to make sure this works with all TS versions from 3.4+. 

Did we consider the implications of simply bumping to TS 4.x?